packages/frontend/src/pages/Recipe.svelte

```diff
@@ -58,17 +58,17 @@ function toggleApplicationDetailsPanel() {
     <Tab title="Models" url="{recipeId}/models" />
   </svelte:fragment>
   <svelte:fragment slot="content">
-    <div class="flex flex-row w-full">
-      <Route path="/" breadcrumb="Summary" >
-        <div class="flex-grow p-5">
+    <div class="grid w-full" style="grid-template-columns: 1fr auto;">
```

Comment:
Just wondering if this a good pratice to use style ?

Reply:
You are right, I used the arbitrary value of tailwinds for grid-template-columns